cmd/internal/cli/actions.go

```diff
@@ -67,6 +72,11 @@ func actionPreRun(cmd *cobra.Command, args []string) {
 		noUmask = true
 		noEval = true
 	}
+
+	// --hostname in OCI mode requires UTS namespace
+	if ociRuntime && (len(hostname) > 0) {
```

Comment:
`--hostname` for non OCI mode will also require a UTS namespace. Are you able to find where it is adding that? 

It'd be good to either...

* Do it at the same level for `--oci`, so it's consistent. -or-
* Do it for the OCI and non-OCI mode in the same place, here.

Reply:
Probably in native/launcher.go?